lms/djangoapps/onboarding_survey/forms.py

```diff
@@ -395,6 +403,34 @@ def save(self, user=None, commit=True):
                 prev_org.save()
 
         extended_profile.organization = organization_to_assign
+
+        is_extended_profile = None
+        try:
+            is_extended_profile = user.extended_profile
+        except:
+            pass
+
+        if not is_extended_profile and extended_profile.org_admin_email:
+            admin_user = None
+
+            try:
+                admin_user = User.objects.get(email=extended_profile.org_admin_email)
+            except:
+                pass
+
+            if admin_user:
+                admin_user.extended_profile.admin_activation_key = uuid.uuid4().hex
+                admin_user.extended_profile.save()
+
+            message_context = {"key": admin_user.extended_profile.admin_activation_key }
+            message_body = render_to_string('emails/admin_activation.txt', message_context)
+
+            with mail.get_connection() as connection:
+                mail.EmailMessage(
+                    "TEST", message_body, settings.ADMIN_ACTIVATION_EMAIL, [extended_profile.org_admin_email],
+                    connection=connection,
+                ).send()
```

Comment:
please move email sending functionality to a separate file and function. Also use the existing function defined to send emails